plugins/module_utils/network/ios/facts/ospfv2/ospfv2.py

```diff
@@ -75,6 +64,17 @@ def populate_facts(self, connection, ansible_facts, data=None):
                 )
 
         for process in current.get("processes", []):
```

Comment:
I'm not sure if we should keep on rendering `passive_interface` (the old param) in facts. Since it is deprecated.

Reply:
We shouldn’t do that as `passive_interface` will still be in effect for the next 4 release cycles.

Reply:
If we keep it, then round trip would fail because of the `mutually_exclusive` clause since the input would have both.